contracts/modules/STO/USDTieredSTO.sol

```diff
@@ -683,7 +670,7 @@ contract USDTieredSTO is USDTieredSTOStorage, STO, ReentrancyGuard {
      * @return uint256[] array of minted tokens in each fund raise type
      */
     function getTokensMintedByTier(uint256 _tier) public view returns(uint256[] memory) {
-        require(_tier < tiers.length, "Invalid tier");
+        //require(_tier < tiers.length, "Invalid tier"); It will revert due to array out of bounds anyway
```

Comment:
I am ok for this. only need to confirm that for any internal call of this function make parent function revert or not as it will lead the `Invalid opcode` error.

Reply:
`Invalid opcode` is revert as well. The only difference is that the error message will say "Invalid opcode" rather than "Invalid tier". It can make debugging a tiny bit more confusing if we at some point hit this error but the chances of us facing an error where we are going over tier limit are next to none.